15-lhcb-dev.md

```diff
@@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
 
 > ## What if getpack asks for my password 1000 times? {.callout}
 > `getpack` might ask you for your password several times.
-> To avoid this, you can create a kerberos token with
-> ```
-> kinit
+> To avoid this, you need to properly configure your `ssh` following the
```

Comment:
I was intentionally vague here, because it's not just public keys. It's public keys with proper GSSAPI configuration... I would leave as is, but maybe somebody more expert than me can suggest the proper terminology.